tests/conftest.py

```diff
@@ -218,6 +218,20 @@ def random_grains(grains_config: dict, random_filters: Filters, tmp_path) -> Gra
     return grains
 
 
+@pytest.fixture
+def small_array_filters(small_array: np.ndarray, filter_config: dict, tmp_path) -> Grains:
```

Comment:
This is returning a `Filters` object not a `Grains`.

```suggestion
def small_array_filters(small_array: np.ndarray, filter_config: dict, tmp_path) -> Filters:
```

Reply:
My bad, must have been overlooked when moving bits around